project/urls.py

```diff
@@ -33,6 +33,7 @@ def trigger_error(request):
     path("users/", include("xshop.users.urls", namespace="users")),
     path("api/", include(api_urlpatterns)),  # API urls from above
     path("", include("xshop.pages.urls", namespace="pages")),
+    path("users/", include("django.contrib.auth.urls")),
```

Comment:
put this line directly under the line 33 since it's more restrictive than the `""` path